app/views/accounts/actions/_edit_action_button.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= link_to path do %>
-  <span class='hide'>
+  <span class='display-none'>
```

Comment:
As discussed at https://github.com/18F/identity-idp/pull/4638/files#r568696179, this is an accessible text alternative meant to provide context for "Edit", "Manage", and "Delete" links.

Suggesting to revert back to a screen-reader-only text:

```suggestion
  <span class='usa-sr-only'>
```